pyprof/parse/kernel.py

```diff
@@ -24,7 +24,12 @@ def demangle(name):
     """
 	Demangle a C++ string
 	"""
-    return cxxfilt.demangle(name)
+    result = name
+    try:
+        result = cxxfilt.demangle(name)
+    except:
+        pass
+    return result
```

Comment:
could you explain:

1. why `cxxfilt.demangle()` seems to fail for some names?
2. why `name` is an acceptable return value in lieu of a failed call to `cxxfilt.demangle()`?

Reply:
Demangle is failing because the name is not properly formatted. I filed a bug with the cutlass team to investigate since it is from their library.
Return the mangled name is just a patch right now. Once the cutlass team gets back to me with the results of the bug investigation, I will update this code.